.github/workflows/codecov.yml

```diff
@@ -33,6 +33,8 @@ jobs:
         elasticsearch-version: 8
         config: |
           http.port: 9200
+        env:
```

Comment:
Need to use the yml config, so I think something like:

config: |
  http.port: 9200
  elastic.password: super-secret

Reply:
The respective configuration setting would be `bootstrap.password`, but this is a “secure” setting, so it cannot be set in the configuration file and rather has to be set in the keystore.

When using the Elasticsearch Docker image, the entrypoint script does exactly that when the container is started. Unfortunately, the `setup-elasticsearch` GitHub action does offer any way to set the bootstrap password and it does not offer any hook for running custom commands between extract ES and starting it either.

So, it looks like we cannot use authentication int the automated test setup and I am going to revert the last commit.